ui/src/audioplayer/PlayerToolbar.js

```diff
@@ -20,15 +21,18 @@ const Toolbar = ({ id }) => {
   return (
     <>
       <GlobalHotKeys keyMap={keyMap} handlers={handlers} allowChanges />
-      <LoveButton
-        record={data}
-        resource={resource}
-        disabled={loading || toggling}
-      />
+      {config.enableFavourites && (
+        <LoveButton
+          record={data}
+          resource={resource}
+          disabled={loading || toggling}
+        />
+      )}
     </>
   )
 }
 
-const PlayerToolbar = ({ id }) => (id ? <Toolbar id={id} /> : <Placeholder />)
+const PlayerToolbar = ({ id }) =>
+  id ? <Toolbar id={id} /> : config.enableFavourites && <Placeholder />
```

Comment:
This check must be done inside the `<Placeholder>` component, as it will have more elements in the future (ex: `<QualityInfo>`)